project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.5.8
+sbt.version=1.7.1
```

Comment:
I checked the [dependency explorer](https://catalogue.tax.service.gov.uk/dependencyexplorer/results?group=org.scala-sbt&artefact=sbt&team=&flag=latest&scope=build&versionRange=%5B0.0.0%2C1.7.1%5D&asCsv=false) and I haven't seen any teams with an sbt version higher than 1.6.2. It might be worth confirming with team-platops that this version of sbt is ok